src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameCheckEnhancedInstanceof.java

```diff
@@ -3,25 +3,29 @@
 
 import java.util.ArrayList;
 import java.util.Locale;
+// config:
+// allowedAbbreviationLength="1"
```

Comment:
Should this be 3, since `testAbbreviationAsWordInNameCheckEnhancedInstanceof` uses the default config?

Reply:
Fixed.